### PR TITLE
Give access to the asymptotic distribution of probability estimate

### PR DESCRIPTION
**Pull request by regislebrun**
This PR provides the following improvements:
+ The DistributionImplementation::computeBilateralConfidenceInterval() method was limited to continuous distributions in the univariate case with no reason.
+ ProbabilitySimulationResult now gives access to the asymptotic distribution of the probability estimate

It fixes issue #1035. It was not possible to implement easily the exact distribution in the case of a crude Monte Carlo simulation due to the way EventSimulation and ProbabilitySimulationAlgorithm interact.

### CODE REVIEW

**Comment by mbaudin47** [comment]
Thanks! The gaussian distribution should do just fine :p !

File: python/test/t_Binomial_std.py [hunk 1]
```diff
@@ -66,6 +66,12 @@
         print("standard moment n=", i, " value=",
               distribution.getStandardMoment(i))
     print("Standard representative=", distribution.getStandardRepresentative())
+    # Confidence interval
+    alpha = 0.05
+    bounds = distribution.computeBilateralConfidenceInterval(1-alpha)
+    lb = bounds.getLowerBound()[0]
+    up = bounds.getUpperBound()[0]
+    print("%.2f%% confidence interval = [%f,%f]" % ((1-alpha)*100,lb,up))
```
[the remark this answers is not in the document]
Comment (jschueller) [reply python/test/t_Binomial_std.py]: maybe just `print(bounds)` is enough

[the remark this answers is not in the document]
Comment (mbaudin47) [reply python/test/t_Binomial_std.py]: Theoretically, it is not a binomial, it is a binomial/n, i.e. a probability, whence the %f. (Furthermore, it is technically a gaussian, not a binomial.)

[the remark this answers is not in the document]
Comment (sofianehaddad) [reply python/test/t_Binomial_std.py]: you are right. Please fix the printing format (%.3f for example)

[the remark this answers is not in the document]
Comment (regislebrun) [reply python/test/t_Binomial_std.py]: We are in t_Binomial_std.py, so it is a Binomial which is tested, not a RandomMixture. And right, `print(bounds)` do the job IMO.

**Pull request merged**